python/foglamp/plugins/north/http_north/http_north.py

```diff
@@ -43,6 +44,16 @@
         'description': 'how long (x seconds) the plugin should wait for pending tasks to complete or cancel otherwise',
         'type': 'integer',
         'default': '10'
+    },
+    "applyFilter": {
+        "description": "Whether to apply filter before processing the data",
+        "type": "boolean",
+        "default": "True"
```

Comment:
Before we check in I think we should make the filter disabled by default

Reply:
I agree, the filter must be disabled.